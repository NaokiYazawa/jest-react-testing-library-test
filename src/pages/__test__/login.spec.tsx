import React from "react";
import { render } from "@testing-library/react";
import Login from "../login";

describe("Home", () => {
  test("Home Page", () => {
    const { container } = render(<Login />);
    expect(container).toMatchSnapshot();
  });
});
