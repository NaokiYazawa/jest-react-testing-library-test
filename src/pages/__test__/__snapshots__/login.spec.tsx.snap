// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home Page 1`] = `
<div>
  <p>
    login page
  </p>
</div>
`;
